docs/change-log.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 ## v.1.7.0
 - Add tests for phone battery features
+- Fill NA with 0 for the selected event features: (1) each sensor's feature extraction script (2) data cleaning script
+- Refactor data cleaning module: update the structure and add dropping highly correlated features section
```

Comment:
Add a data cleaning module to replace NAs with 0 in selected event-based features, discard unreliable rows and columns, and discard columns with zero variance.

Reply:
Hi @JulioV , we also drop highly correlated features. Shall we update it as follows?

- Add a data cleaning module to replace NAs with 0 in selected event-based features, discard unreliable rows and columns, discard columns with zero variance, and discard highly correlated columns.